frontend/src/components/ShootWorkers/GVolumeSizeInput.vue

```diff
@@ -26,6 +42,10 @@ export default {
     modelValue: {
       type: [String, Number],
     },
+    customStorageSize: {
```

Comment:
The name of the parameter says it is a size which would mean the value has type number. Rename to `hasCustomStorageSize`.